internal/app/singularity/cache_clean_linux.go

```diff
@@ -187,8 +186,7 @@ func CleanSingularityCache(cleanAll bool, cacheCleanTypes []string, cacheName st
 			return err
 		}
 		if !foundMatch {
-			sylog.Warningf("No cache found with given name: %v", cacheName)
-			os.Exit(0)
+			return fmt.Errorf("No cache found with given name: %v", cacheName)
```

Comment:
Umm, this will exit with a non-0. I think it would be better if you do:

```go
			sylog.Warningf("No cache found with given name: %s", cacheName)
			return nil
```

Because if a user just wants to make sure that cache is cleared, without failing in there script.

Reply:
fixed